extensions/nns/e2e/tests/nns.bash

```diff
@@ -22,33 +22,27 @@ teardown() {
 }
 
 @test "ic-nns-init binary exists and is executable" {
-    dfx cache install
```

Comment:
So with this change in the e2e test we can be sure that the extensions aren't accidentally using the dfx bundled binaries?

Reply:
I removed `dfx cache install`s because, for some reason, it was overwriting the content of dfx cache directory, and subsequently, it was deleting the `extensions` directory, effectively removing the `nns` extension that was installed during `setup()`. Not sure what's the reasoning behind it, but this is indeed what it does:
- `dfx cache install`: https://github.com/dfinity/sdk/blob/master/src/dfx/src/commands/cache/install.rs#L12 
- https://github.com/dfinity/sdk/blob/master/src/dfx/src/config/cache.rs#L37
- here it deletes the dir: https://github.com/dfinity/sdk/blob/master/src/dfx/src/config/cache.rs#L141
---
@dfinity/sdk perhaps we should add `--force` flag to `dfx cache install`, so that users won't wipe their installed extensions when trying to fix their dfx cache?